networking/network_observability/network-observability-overview.adoc

```diff
@@ -0,0 +1,47 @@
+:_content-type: ASSEMBLY
+[id='network-observability-overview']
+= About Network Observability
+include::_attributes/common-attributes.adoc[]
+:context: network-observability-overview
+
+toc::[]
+
+Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters.
+
+[id="dependencies-network-observability"]
+== Dependencies of Network Observability Operator
+
+The Network Observability Operator requires the following Operators:
+
+* Loki Operator: You must install Loki install before you install the Network Observability Operator. The Loki Operator is the backend that you can use to store all collected flows.
+* Grafana Operator: You can optionally install Grafana Operator for using custom dashboards and querying capabilities.
+* Kafka Operator: You can optionally install Kafka Operator. The Kafka Operator provides scalability, resiliency and high availability in the {product-title} cluster.
+
+[id="network-observability-operator"]
+== Network Observability Operator
+
+The Network Observability Operator deploys a monitoring pipeline to collect and enrich network flows. The NetObserv eBPF agent or CNI can create network flows. You can export these network flows in IPFIX format, such as OVN-Kubernetes.
```

Comment:
```suggestion
The Network Observability Operator deploys a monitoring pipeline to collect and enrich network flows.
```

I would avoid these eBPF / IPFIX mention at this stage:
- we only support eBPF agent downstream
- you can use IPFIX export with OVN Kubernetes only and without RH support

What do you think ?

Reply:
Addressed